packages/manager/apps/pci-instances/src/pages/instances/create/hooks/useForm.ts

```diff
@@ -64,10 +64,19 @@ export const useForm = (projectId: string) => {
   const distributionImageTypeDefaultValue =
     mockedDistributionImageType[0]?.value ?? null;
 
-  const distributionImageNameDefaultValue =
-    mockedDistributionImageList.find(
-      ({ type }) => type === distributionImageTypeDefaultValue,
-    )?.id ?? null;
+  const distributionImage = mockedDistributionImageList.find(
+    ({ type }) => type === distributionImageTypeDefaultValue,
+  );
+
+  const distributionImageNameDefaultValue = distributionImage?.id ?? null;
+
+  const flavorImagePriceDefaultValue = distributionImage?.pricing?.hour ?? null;
+
+  const distributionImageVersionDefaultValue =
+    distributionImage?.versions?.[0]?.value ?? null;
+
+  const distributionLicencePrice =
+    distributionImage?.pricing?.licence?.total ?? null;
```

Comment:
Suggestion:
I see this in at least three different files (DistributionImageList.component.tsx, DistributionImageType.component.tsx and this one here).
So I think you could introduce some king of method like 'getDistributionLicensePrice = (distributionImage) => ...'